app/javascript/packs/braintree_customer_form.ts

```diff
@@ -16,24 +16,21 @@ document.addEventListener('DOMContentLoaded', async () => {
     throw new Error('The target ID was not found: ' + CONTAINER_ID)
   }
 
-  const { clientToken, threeDSecureEnabled, formActionPath, countriesList, selectedCountryCode } = container.dataset as Record<string, string>
+  const data = safeFromJsonString<BraintreeFormDataset>(container.dataset.braintreeForm)
 
-  const billingAddress = safeFromJsonString<BillingAddressData>(container.dataset.billingAddress) ?? {}
-  for (const key in billingAddress) {
-    // @ts-expect-error FIXME
-    if (billingAddress[key] === null) {
-      // @ts-expect-error FIXME
-      billingAddress[key] = ''
-    }
```

Comment:
This has been removes in favor of server side `transform_values { |value| value.presence || '' }`.